modules/Makefile.circleci

```diff
@@ -45,6 +45,8 @@ circle\:tag-latest:
 	@$(SELF) docker:login
 	@echo "INFO: Tagging latest"
 	@$(SELF) DOCKER_TAG=latest docker:push
+	@git tag --force "$(CIRCLE_BRANCH)-docker-latest"
```

Comment:
If the docker:push command above fails, will it still proceed to this step? If so we probably want to check and not apply the tag if the push failed.